common/encfstools.py

```diff
@@ -511,7 +511,7 @@ def __init__(self, cfg, string = True):
             .replace('Schnappschuss', '(?:Schnappschuss|Snapshot)')
 
         #precompile some regular expressions
-        host, port, user, path, cipher = cfg.sshHostUserPortPathCipher()
+        host, user, path = cfg.sshHostUserPortPathCipher()
```

Comment:
This won't work. Have look into `sshHostUserPortPathCipher()` and what it returns. It still returns a tuple with 5 values, not 3.

In cases like this it is usual to use `_` as a variable name to indicate that this value will be ignored and not used. **But** in our case the name `_` is reserved by the function `_()` and used for translating strings with the GNU Gettext toolkit. Because of that it is an alternative to use the `_` plus a name like this.

```suggestion
        host, _post, user, path, _cipher = cfg.sshHostUserPortPathCipher()
```